_includes/sidenav.html

```diff
@@ -24,16 +23,17 @@
 
 <nav class="nav-secondary expandable {{ additional_classes }}">
     <h1 class="u-visually-hidden">
-    {%- for href, id, caption in items %}
+    {%- for item in items %}
```

Comment:
Ah, good point. Makes me realize we need a fallback, too, if an index item is not present. E.g., the FinEd Resources.